spec/cyanobyte.schema.json

```diff
@@ -0,0 +1,216 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://github.com/cyanobyte/cyanobyte.schema.json",
```

Comment:
This would not be the canonical url.

Reply:
What would it be? Just threw something in there temporarily.

Reply:
`https://github.com/google/cyanobyte/spec/cyanobyte.schema.json` would end up being the destination URL